wallet/node/Main.hs

```diff
@@ -71,27 +68,26 @@ pluginsGT WalletArgs {..}
     | otherwise = []
 
 action :: WalletNodeArgs -> Production ()
-action (WalletNodeArgs (snArgs@SimpleNodeArgs {..}) (wArgs@WalletArgs {..})) = do
+action (WalletNodeArgs (cArgs@CommonNodeArgs {..}) (wArgs@WalletArgs {..})) = do
     systemStart <- CLI.getNodeSystemStart $ CLI.sysStart commonArgs
     logInfo $ sformat ("System start time is " % shown) systemStart
     t <- currentTime
     logInfo $ sformat ("Current time is " % shown) (Timestamp t)
-    currentParams <- getSimpleNodeParams snArgs systemStart
-    putText $ "Running using " <> show (CLI.sscAlgo commonArgs)
+    currentParams <- getNodeParams cArgs systemStart
     putText $ "Wallet is enabled!"
 
     let vssSK = fromJust $ npUserSecret currentParams ^. usVss
-    let gtParams = gtSscParams snArgs vssSK
+    let gtParams = CLI.gtSscParams cArgs vssSK
 
     let sscParams :: Either (SscParams SscNistBeacon) (SscParams SscGodTossing)
-        sscParams = bool (Left ()) (Right gtParams) (CLI.sscAlgo commonArgs == GodTossingAlgo)
+        sscParams = bool (Left ()) (Right gtParams) False
```

Comment:
It's really confusing. You can remove this `sscParams` completely and just write `actionWithWallet gtParams currentParams wArgs`.